firefox-ios/Shared/Strings.swift

```diff
@@ -2716,6 +2716,12 @@ extension String {
             value: "Loading ...",
             comment: "The label shown while loading a document in the web view's custom document loading UI"
         )
+        public static let DocumentLoadingAccessibilityLabel = MZLocalizedString(
+            key: "WebView.DocumentLoadingAccessibilityLabel.v137",
+            tableName: "WebView",
+            value: "Loading Document",
+            comment: "The accessibility label associated with Document loading label"
```

Comment:
Translators don't necessarily have the string in the same order we see them here I think. Its best to give them full context each time.
```suggestion
            comment: "The accessibility label read when loading a document in the web view's custom document loading UI."
```